views/partials/bunnyDetails.handlebars

```diff
@@ -0,0 +1,10 @@
+<div class="post">
+  <div class="post-contents">
+    <div>
+      <img src="/img/infants/{{image}}" alt="{{description}}">
+    </div>
+    <div>
+      <a href="#" class="post-title">{{description}}</a> <span class="post-price">${{price}}</span>
+    </div>
+  </div>
+</div>
```

Comment:
This should use different CSS classes than the list so we can apply CSS styles easier.